### ISSUE TITLE
myaddr.io and all subdomains blocked by Google Safe Browsing

### ISSUE THREAD

**Issue by orazioedoardo** [brianshea2/addr.tools#12]
See: https://transparencyreport.google.com/safe-browsing/search?url=https:%2F%2Fmyaddr.io&hl=en. At the time of writing it’s reported as unsafe. Due to Safe Browsing being adopted by multiple browsers, this makes all DDNS under myaddr.io widely inaccessible. It may be due to a specific site, don’t know if you can report as false positive.

**Comment by brianshea2** [comment]
Yes, I'm aware and have made the report to remove the domain. Seems to have started within the last 24 hours. Unfortunately, a few user-controlled subdomains were being used for phishing. I received abuse complaints for those subdomains and immediately suspended them. I'm assuming this is related. Unfortunately, we are small fish, at the mercy of Google at the moment. The other domains seem unaffected. Growing pains of the service.... I'm hoping to get the domains added to the [PSL](https://github.com/publicsuffix/list) soon, and that may help in the future to prevent the whole domain from being blocked.

Will keep this issue open and provide updates until the domain is no longer considered unsafe.

**Comment by orazioedoardo** [comment]
Could be this? https://crt.sh/?id=15680215270

**Comment by brianshea2** [comment]
I won't go as far as to list the subdomains that were reported, but I will say those subdomains are no longer active and have been blocked from future re-use.

**Comment by orazioedoardo** [comment]
> I'm hoping to get the domains added to the [PSL](https://github.com/publicsuffix/list) soon, and that may help in the future to prevent the whole domain from being blocked.

You could use that as a justification on why to add myaddr to the PSL. They argued the service it small but obviously it won’t grow if one bad apple takes down everything. Having it on a PSL is also a security feature because it makes a.myaddr.io and b.myaddr.io cross-site.

By the way, do you accept donations? If so you could add options in the README.

**Comment by brianshea2** [comment]
I'll be opening a new PR to get on the PSL soon, and I'll reference this. The user base is much larger now than my original attempt to get on the PSL, so I'm hoping it goes better this time around.

And thank you for asking! I have nothing set up to accept donations at this time. I run these services because I find it fun, and the costs are very minimal for now.

**Comment by brianshea2** [comment]
Looks like it's no longer blocked.
